.github/pull_request_template.md

```diff
@@ -8,6 +8,10 @@ Fixes #[insert GH or internal issue number(s)].
   - [ ] TODO (before merge)
   - [ ] Included
   - [ ] Not necessary because:
+- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
+  - [ ] I don't know
+  - [ ] Required / Maybe required
```

Comment:
This is useful for internal and external contributors.

For internal contributors, this will serve as a reminder to add the "e2e" label before the PR is merged. Especially when you don't want to run them while the PR is in-flight to save runners.

For external contributors, they won't be able to run the e2e tests themselves (due to security implications). This will serve as a reminder to add the "e2e" label and also for us, as maintainers, to run the e2e tests in a separate PR (only once we have reviewed the changes and confirmed no malicious intent)